tidb-server/main.go

```diff
@@ -74,6 +74,7 @@ const (
 	nmConfigStrict     = "config-strict"
 	nmStore            = "store"
 	nmStorePath        = "path"
+	nmTempStoragePath  = "temp-storage-path"
```

Comment:
```suggestion
	nmTempStoragePath  = "tmp-storage-path"
```

Reply:
Should change nmTempStoragePath to nmTmpStoragePath as well?